virt/virt-using-the-cli-tools.adoc

```diff
@@ -14,8 +14,16 @@ The two primary CLI tools used for managing resources in the cluster are:
 
 * You must xref:../virt/install/virt-installing-virtctl.adoc#virt-installing-virtctl[install the `virtctl` client].
 
-include::modules/virt-virtctl-commands.adoc[leveloffset=+1]
 include::modules/virt-openshift-client-commands.adoc[leveloffset=+1]
 
 For more comprehensive information on `oc` client commands, see the
 xref:../cli_reference/openshift_cli/developer-cli-commands.adoc#cli-developer-commands[{product-title} CLI tools] documentation.
+
+include::modules/virt-virtctl-commands.adoc[leveloffset=+1]
+include::modules/virt-creating-pvc-with-virtctl-guestfs.adoc[leveloffset=+1]
+include::modules/virt-about-libguestfs-tools-virtctl-guestfs.adoc[leveloffset=+1]
+
+[id="additional-resources_virt-libguestfs-tools-virtctl-guestfs"]
+== Additional resources
+
+* See link:https://libguestfs.org[Libguestfs: tools for accessing and modifying virtual machine disk images] for additional information about using `libguestfs` tools.
```

Comment:
The contribution guide encourages us to simplify link text. I'd make this just:
`* link:https://libguestfs.org[Libguestfs: tools for accessing and modifying virtual machine disk images]`